api_tests/test_network.py

```diff
@@ -52,6 +52,10 @@ class TestSocketCreate(TestCase):
     def test_deny_create_socket(self, address_family, socket_type):
         """
         Can't create a socket (even without calling bind or connect).
+
+        If we *could* create sockets, the next step would be checking which
+        destinations we can bind or connect to. In particular, we'd want to
+        check each of public, network-internal, and loopback addresses.
```

Comment:
It's unclear who the _we_ is, and what is being stated here. Are you saying that someone could have this test fail on a safe system, but they would need to manually verify certain things? Or are you saying that this test might end up being overly restrictive, and you are explaining some of what we'd need to do to tune this test? Or some combo?

Reply:
This is basically some notes on an implementation detail of the tests.

When I wrote the tests initially, I expected to be able to *create* the sockets, but then be denied *using* the sockets. The idea was going to be that I would try connecting to various types of IP addresses in various network segments. But as it turned out, the denial happened much earlier in the code than I expected! The result is that these tests now have no "coverage" of IPv4 vs. IPv6, internal vs. public, etc.

It feels weird that way, and it makes me a little nervous about what happens if the system needs to later evolve to support creating sockets but not bind/connecting them; we'd be missing coverage. So this note is really to anyone who comes along later, describing how the test would need to evolve.

Reply:
This proposal is to more clearly provide the context for this comment:
```
If we later decide to allow creating sockets, the next steps would be
enhancing these tests to check which destinations we can bind or
connect to. In particular, we'd want to check each of public, 
network-internal, and loopback addresses.
```

Reply:
Yeah, I'll update it to give more context, something like that.